src/services/IPFSService.ts

```diff
@@ -134,6 +134,7 @@ export default class IPFSService {
       await sleep(1000);
       const ipfsContent = await this.getContentFromIPFS(hash);
       console.debug('[IPFS CONTENT]', ipfsContent);
+      // eslint-disable-next-line
       if (content === content) uploaded = true;

```

Comment:
I think this is actually a bug? Should this be `ipfsContent === content`?

Reply:
Make sense!